ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -138,9 +138,7 @@ Item {
                                             suggestionsListView.incrementCurrentIndex()
                                         }
                                     } else {
-                                        if (event.key === Qt.Key_Backspace && edit.text === "") {
-                                            root.entryRemoved(listView.itemAtIndex(listView.count - 1))
-                                        } else if (event.key === Qt.Key_Return || event.key === Qt.Enter)  {
+                                        if (event.key === Qt.Key_Return || event.key === Qt.Enter)  {
```

Comment:
AFAIR remove entry by backspace was desired, also linked issue says nothing about removing it. Could you please clarify it with the design team?

Reply:
updated